src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,228 @@
+
+@DisplayName("지하철 노선 관련 기능")
+@Sql({"/sql/station-data.sql"})
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+    */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+
+        // given
+        LineRequest lineRequest = new LineRequest();
+        lineRequest.setName("신분당선");
+        lineRequest.setColor("red");
+        lineRequest.setUpStationId(1);
+        lineRequest.setDownStationId(2);
+        lineRequest.setDistance(10);
+
+
+        // when
+        // then
+        assertThat(createLine(lineRequest).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+    }
+
+    /**
+     * Given: 여러 개의 지하철 노선이 등록되어 있고,
+     * When: 관리자가 지하철 노선 목록을 조회하면,
+     * Then: 모든 지하철 노선 목록이 반환된다.
+     */
+    @DisplayName("지하철 노선 목록을 조회한다.")
+    @Test
+    void showLines() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        LineRequest lineRequestGreen = new LineRequest();
+        lineRequestGreen.setName("분당선");
+        lineRequestGreen.setColor("bg-green-600");
+        lineRequestGreen.setUpStationId(1);
+        lineRequestGreen.setDownStationId(3);
+        lineRequestGreen.setDistance(20);
+
+        assertThat(createLine(lineRequestGreen).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // when
+        List<String> lineNames =
+                given().log().all()
+                        .when().get("/lines")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        //then
+        assertThat(lineNames).contains("신분당선", "분당선");
+    }
+
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 조회하면,
+     * Then: 해당 노선의 정보가 반환된다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void showLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        // when
+        List<String> lineNames =
+                given().log().all()
+                        .when().get("/lines")
+                        .then().log().all()
```

Comment:
```suggestion
                        .then().log().all()
                        .statusCode(HttpStatus.OK.value())
```

이렇게 응답코드에 대한 검증도 해주면 어떨까요? 😄

Reply:
넵 추가하겠습니다~